.github/workflows/pr-labeler.yml

```
@@ -1,15 +1,16 @@
   pull_request_target:
     types: [opened, synchronize]
 
 jobs:
   triage:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
     steps:
-      - uses: actions/labeler@v5
+      - uses: actions/labeler@v4
```

actionlint (1.7.7)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)